CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+## Pelion Edge utilities 2.5.0
```

Comment:
Should this match the 2.0.5 GitHub label on this PR?

Reply:
Ohh yeah.. Thanks for catching this.